getting-started/02-new-cif-project.md

```diff
@@ -0,0 +1,394 @@
+
+The [CIF Project archetype](https://github.com/adobe/aem-cif-project-archetype) creates a minimal Adobe Experience Manager (AEM) CIF project as a starting point for customer projects using [CIF Core Components](https://github.com/adobe/aem-core-cif-components). The CIF Project archetype should be used for new projects looking to integrate AEM and Magento using the Commerce Integration Framework. The CIF Project archetype can also serve as a useful reference for existing AEM projects looking to add the CIF related dependencies.
+
+In this guide we will generate a new AEM CIF project using the CIF Project archetype and explore how to use it.
+
+## Background
+
+**What is a Maven project?** - [Apache Maven](https://maven.apache.org/) is a software management tool to build projects. All AEM implementations use Maven projects to build, manage and deploy custom code on top of AEM.
+
+**What is a Maven archetype?** - A [Maven archetype](https://maven.apache.org/archetype/index.html) is a template or pattern for generating new projects. The CIF Project archetype allows us to generate a new project with a custom namespace and include a project structure that follows best practices.
+
+## Prerequisites
+
+The following tools and technologies are required:
+
+* [Java 1.8](https://www.oracle.com/technetwork/java/javase/downloads/index.html) or [Java 11](https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html) (AEM 6.5+ only)
+* [Apache Maven](https://maven.apache.org/) (3.3.9 or newer)
+* Adobe Experience Manager
+  * [AEM 6.5](https://helpx.adobe.com/experience-manager/6-5/sites/deploying/using/technical-requirements.html)
+  * [AEM 6.4.4+](https://helpx.adobe.com/experience-manager/6-4/release-notes/sp-release-notes.html)
+
+It is also considered a best practice to include the Adobe Public Maven Repository in your Maven settings. This file, commonly found at `~/.m2/settings.xml` can be updated following this [Knowledge Base](https://helpx.adobe.com/experience-manager/kb/SetUpTheAdobeMavenRepository.html) article.
+
+## Generate the project
+
+The next series of steps will take place using a UNIX based command line  terminal,  but should be similar if using a Windows terminal.
+
+1. Verify that Maven is correctly installed and available from the command line:
+
+    ```shell
+    $ mvn -version
+      Apache Maven 3.3.9
+      Maven home: /Library/apache-maven-3.3.9
+      Java version: 11.0.2, vendor: Oracle Corporation
+      Java home: /Library/Java/JavaVirtualMachines/jdk-11.0.2.jdk/Contents/Home
+    ```
+
+2. Navigate to a directory, i.e a source folder, in which you want to generate the project:
+
+    ```shell
+    $ cd ~/src
+    ```
+
+3. Copy+Paste the following command to initiate the creation of a new project in interactive mode:
+
+    ```shell
+    mvn archetype:generate \
+    -DarchetypeGroupId=com.adobe.commerce.cif \
+    -DarchetypeArtifactId=cif-project-archetype \
+    -DarchetypeVersion=0.2.0
```

Comment:
Should we replace `0.2.0` with `x.x.x` or similar here? Otherwise, this must be updated on every release.

Reply:
sounds good!